doc/src/explore/devnet.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Experiment with Sui DevNet
+---
+
+Welcome to the beginnings of the Sui DevNet. It exists now to gain operational experience with the Sui software in a public setting. The Sui DevNet currently consists of:
+
+* A four-validator network with all nodes operated by Mysten Labs. Clients send transactions and read requests via this endpoint: http://gateway.devnet.sui.io/
+* A public network [Sui Explorer](https://github.com/MystenLabs/sui/tree/explorer-rest/explorer) for browsing the TestNet transaction history: http://explorer.devnet.sui.io
+* A [Discord channel](https://discordapp.com/channels/916379725201563759/971488439931392130) for requesting test coins that can be used to pay for gas on the test network. These coins have no financial value and will disappear each time we reset the network.
+
+Many improvements to the Sui DevNet are underway, such as the ability to run FullNodes and use a browser-based wallet. See the Sui DevNet blog post announcement for full details on upcoming features.
```

Comment:
Should we just say "full nodes" instead of "FullNodes"?

Reply:
Sure! Whatever works for you all. I got this from:
https://www.notion.so/mystenlabs/Fullnodes-in-the-Sui-Ecosystem-b28f6a2b0a6c4fdd86f1ea852ab49ac2

And tried to emulate:
https://aptos.dev/basics/basics-fullnodes/

Reply:
Done!